### ISSUE TITLE
REST API /wc/store/cart* endpoint documentation needed - Headless WooCommerce

### ISSUE THREAD

**Issue by thd-fox** [woocommerce/woocommerce#27032]
I searched for a long time on the internet and also here in the existing issues.
Though I found nothing about the cart rest endpoint.
I'm going to build a headless WooCommerce client.
I'm running against various errors that I need to get rid of by reverse engineering the woocommerce code.

I found a plugin that provides a payed solution but I don't want to rely on it as there are obvious bugs found.

**Describe the solution you'd like**
I'd love to see the endpoint /wc/store/cart* documented in the woo rest api documentation

**Comment by seb86** [comment]
Have you had a [look at CoCart](https://github.com/co-cart/co-cart) @thd-fox It's a REST API designed to build a headless WC store.

**Comment by thd-fox** [comment]
@seb86 yes I already had a look on this plugin. But why use a plugin if there already is this native woo api endpoint. I don't want to have an extra dependency of a payed plugin. If I can't figure it out I will need to use this plugin. Thank you for your suggestion. :)

**Comment by seb86** [comment]
CoCart was built because there was no native API at the time. Plus the new routes are designed for internal use only for the cart and checkout blocks. Not for headless purposes.

**Comment by juliaamosova** [comment]
Hi @thd-fox,

Thank you for reporting the issue. WooCommerce REST API is being developed in another repository: https://github.com/woocommerce/woocommerce-rest-api. 

I am going to move your bug report there so that we could reply at the right place. 

However, please note that we currently don't have a cart endpoint in the WooCommerce REST API.
